core/trino-main/src/main/java/io/trino/sql/planner/ConnectorExpressionTranslator.java

```diff
@@ -210,6 +232,36 @@ public ConnectorToSqlExpressionTranslator(Session session, PlannerContext planne
             return Optional.of(builder.build());
         }
 
+        private Optional<Expression> translateIsNotNull(ConnectorExpression argument)
+        {
+            Optional<Expression> translatedArgument = translate(argument);
+            if (translatedArgument.isPresent()) {
+                return Optional.of(new IsNotNullPredicate(translatedArgument.get()));
+            }
+
+            return Optional.empty();
```

Comment:
```suggestion
            return translatedArgument.map(translatedArg -> new IsNotNullPredicate(translatedArg));
```

Reply:
Same in `translateIsNull`

Reply:
It's not more readable for me. I'll leave it as it is